### ISSUE TITLE
Cross-gender option.

### ISSUE THREAD

**Issue by Tburgos** [lushen124/Universal-FE-Randomizer#51]
I don't have any real knowledge with coding persay so hear this out and sorry if i'm hard to understand.
The option to let the player decide if they are ok with males being female classes and vice versa would be very appreciated. In the old randomizer there was an option to let you click it if you want that but not clicking cross gender keeps it sensical. For example rutger couldn't become a pegasus knight. And lyn can't be a pirate. Thank you for understanding.

**Comment by lushen124** [comment]
Nah, it's not that difficult to do. It's just unfortunate that FE7 has a surprisingly limited class pool for female classes, but it's not difficult to do (considering the last randomizer did this). 🙂
